eve/scripts/launch-end2end.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+# Helper script that check if the content and name of the branch is documentation related only.
+# This help us deciding whether we should or not launch the end2end tests.
+# Returns true when it's not documentation related and false when it is. 
+
+
+if git rev-parse --abbrev-ref HEAD | grep -q 'documentation/*' ; then
+    diff="$(git diff --name-only HEAD~1 | grep -v docs/)"
+    if [ ! -z "$diff" ]; then
+        echo "true"
```

Comment:
Not sure the quotes are required but I do not know how eve handles this.

Reply:
They're not, I can remove them

Reply:
done